api/v1/submissions/PKPSubmissionController.php

```diff
@@ -111,7 +112,10 @@ class PKPSubmissionController extends PKPBaseController
         'getPublicationIdentifierForm',
         'getPublicationLicenseForm',
         'getPublicationTitleAbstractForm',
-        'getChangeLanguageMetadata'
+        'getChangeLanguageMetadata',
+        'getJAVStageMetadata',
```

Comment:
There's inconsistency between these two lines on `JAV` vs. `Jav`; I don't have a strong preference but they should be consistent. In any case, I think it would be better to avoid the acronym, as most won't be familiar with it. And listing stage and numbering in the API name is overspecific. Instead, maybe `changeVersionData`?